Makefile

```diff
@@ -118,6 +129,16 @@ clone: # Clone REANA source code repositories locally.
 
 build: # Build REANA client and cluster components.
 	@echo -e "\033[1;32m[$$(date +%Y-%m-%dT%H:%M:%S)]\033[1;33m reana:\033[0m\033[1m make build\033[0m"
+ifeq ($(BUILD_TYPE),release)
+	make release-build
+else ifeq ($(BUILD_TYPE),development)
+	make dev-build
+else
+	echo "Unknown build type $$BUILD_TYPE"
+endif
+
+dev-build: # Build REANA client and cluster components.
```

Comment:
Do not document `dev-build` target so that it would not appear in the help when people type `make`.